### PR TITLE
Add payment_completed timestamp to paid registrations on hover

### LINKED ISSUE #6326: Add time to Registered and paid on dates.
**Is your feature request related to a problem? Please describe.**
With Registrations filling up extremely fast, it would be beneficial to know the exact time someone registered and/or paid in addition to the date. Currently for the "Paid on" column only the date is shown

**Describe the solution you'd like**
Have a date + time show instead on the `/edit/registrations` page.

### PR DESCRIPTION
**Pull request by saranshgrover**
Closes #6326

### CODE REVIEW

File: WcaOnRails/app/helpers/registrations_helper.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def please_sign_in(message_key, comp, args = {})
   def registration_date_and_tooltip(competition, registration)
     if @competition.using_stripe_payments?
       [registration.last_payment_date&.to_date || I18n.t('registrations.list.not_paid'),
-       I18n.t('registrations.list.payment_requested_on', date: registration.created_at)]
```
Comment (saranshgrover) [inline WcaOnRails/app/helpers/registrations_helper.rb:39]: nit - is there a better way to handle this without a really ugly ternary?

Comment (viroulep) [reply WcaOnRails/app/helpers/registrations_helper.rb:39]: The only alternative I can think of is to do something like this in the `if`:
```ruby
tooltip_message = if registration.last_payment_date
                    I18n.t('registrations.list.payment_completed_on', date: registration.last_payment_date)
                  else
                    I18n.t('registrations.list.payment_requested_on', date: registration.created_at)
                  end
```
and return the tooltip message.
But honestly the ternary doesn't look that bad to me.

**Review by viroulep — COMMENTED** [review]
Looks pretty clean to me :)

File: WcaOnRails/app/helpers/registrations_helper.rb (hunk 1 above)
Comment (gregorbg) [reply WcaOnRails/app/helpers/registrations_helper.rb:39]: I've seen uglier ternaries by all means :D
What you can do is try to make it look pretty by changing the indentation, but that is a super-nitpick and should not block this PR. Using a ternary expression per se is totally fine for this code.

**Review by gregorbg — APPROVED** [review]
Overall short'n'sweet implementation!
Also, no worries about the ternary expression :)

**Comment by saranshgrover** [comment]
Can this be merged?

**Comment by gregorbg** [comment]
Absolutely! :ship:

**Pull request merged**